src/Lesson_twalve/AnalyzerText.java

```diff
@@ -0,0 +1,90 @@
+package Lesson_twalve;
+
+import java.io.*;
+import java.util.*;
+
+public class AnalyzerText {
+
+
+    //*****  method of reading text line by line
+    private static String readText(File fileBook) {
+        StringBuilder textBook = new StringBuilder();
+        try (BufferedReader readText = new BufferedReader(new FileReader(fileBook))) {
+            String line;
+            while ((line = readText.readLine()) != null) {
+                textBook.append(line).append(" ");
+            }
+        } catch (IOException e) {
+            System.out.println("Помилка читання книги " + fileBook);
+        }
+        return textBook.toString();
+    }
+
+
+    //***** method for getting words longer than 2 characters
+    private static Map<String, Integer> analyzeText(String text) {
+        Map<String, Integer> wordFrequencyMap = new HashMap<>();
+        StringTokenizer stringToken = new StringTokenizer(text, " \t\n\r\f.,;:\"");
+        while (stringToken.hasMoreTokens()) {
+            String word = stringToken.nextToken().toLowerCase();
+            if (word.length() > 2) {
+                wordFrequencyMap.put(word, wordFrequencyMap.getOrDefault(word, 0) + 1);
+            }
+        }
+        return wordFrequencyMap;
+    }
+
+    //*****   Writing the result to a file
+    private static void writeBookStatistics(String bookName, List<String> topWords, int totalUniqueWords) {
+        String fileName = "src/" + bookName + "_statistic.txt";
+        try (PrintWriter writer = new PrintWriter(new FileWriter(fileName))) {
+            topWords.forEach(writer::println);
+            writer.write("\nЗагальна кількість слів: " + totalUniqueWords);
+        } catch (IOException e) {
+            System.out.println("Помилка запису до файлу статистики " + fileName);
```

Comment:
as we discussed on the lesson, errors need  to be handled, not just printed, all text need to be in English